README.md

```diff
@@ -35,26 +35,23 @@ and provides a utility that gets tests up and running quickly
     process.env.NODE_ENV = 'test'
     ```
 
-  * writes to `mycha.coffee`
+  * writes to `mycha.yml`
     ```coffee
-    module.exports =
+    # Environment variables to add to process.env when running mocha
+    # mochaEnv:
 
-      # Environment variables to add to process.env when running mocha
-      mochaEnv: {}
+    # Default options to pass to mocha (can be overriden by command line options)
+    mochaOptions:
+      colors: true
+      compilers: 'coffee:coffee-script/register'
```

Comment:
removed quotes from all strings except testFilePattern as `*` has a special meaning in yaml.